db/seeds.rb

```diff
@@ -32,16 +32,11 @@
   + 840  # line islands
 ]
 
-### USERS
-
-Allocation.destroy_all
-Bucket.destroy_all
-Comment.destroy_all
-Contribution.destroy_all
 Group.destroy_all
-Membership.destroy_all
 User.destroy_all
```

Comment:
I'd look in to the rake db:reset command. It runs

    rake db:drop
    rake db:create
    rake db:schema:load
    rake db:seed

If you use this, you shouldn't need to do any destroy_all steps in your seeds file. And I would avoid using destroy_all in your seeds file, since it's not a typical behaviour of this file (it should really just add records)